# Request

# Target acceleration
geometry_msgs/Accel acceleration

---
# Response

bool success
